Core/src/Material/Interactions.cpp

```diff
@@ -114,19 +115,19 @@ inline float logDeriveEpsilon(float qOverP, const RelativisticQuantities& rq) {
 }
 
 /// Compute the density correction factor delta/2.
-inline float computeDeltaHalf(float meanExitationPotential,
+inline float computeDeltaHalf(float meanExcitationEnergy,
                               float molarElectronDensity,
                               const RelativisticQuantities& rq) {
-  /// Uses RPP2018 eq. 33.6 which is only valid for high energies.
+  // Uses RPP2018 eq. 33.6 which is only valid for high energies.
   // only relevant for very high ernergies; use arbitrary cutoff
   if (rq.betaGamma < 10.0f) {
     return 0.0f;
   }
   // pre-factor according to RPP2019 table 33.1
   const float plasmaEnergy =
-      PlasmaEnergyScale * std::sqrt(1000.f * molarElectronDensity);
+      PlasmaEnergyScale * std::sqrt(molarElectronDensity / (1 / 1_cm3));
```

Comment:
I guess this is on purpose not `molarElectronDensity * 1_cm3`?

Reply:
I think my thought here was that `molarElectronDensity` is expected to be in $cm^{-3}$ here. The way we usually convert these units is then to divide it by our native unit constants. Does that make sense to you? Happy to change it